src/main/java/core/basesyntax/model/zoo/Animal.java

```diff
@@ -1,9 +1,31 @@
 package core.basesyntax.model.zoo;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.persistence.Table;
+
+@Entity(name = "animals")
+@Table(name = "aminals")
```

Comment:
some

Reply:
not sure

Reply:
I am also not sure. Seems like Enity name = alias for class for Hibernate. But table name is name of table.